### PR TITLE
minor node fixups

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**
These changes are inspired by deno, but all of them should be uncontroversial "best practice" type stuff.

 - [x] someone please force-push a `node_modules` update.  My connection is too slow :sweat_smile:

### CODE REVIEW

**Comment by allisonkarlitskaya** [comment]
fwiw, the commit messages mention `--allow-all`, but this is closer to the actual required set:

```sh
deno run \
    --allow-env \
    --allow-read \
    --allow-sys=cpus \
    --allow-write=dist,qunit \
    --allow-run=tools/node-modules,node_modules/.deno/@esbuild+linux-x64@0.23.0/node_modules/@esbuild/linux-x64/bin/esbuild \
    build.js 
```

**Review by martinpitt — APPROVED** [review]
Thanks!

**Comment by allisonkarlitskaya** [comment]
The timeout in the CDP code has been happening from time to time since the last little while.

I'm not sure what to do about the stylelint failure, but the three possibilities:
 - fix the imports
 - ignore the rule in stylelintrc
 - pin the old version

Someone who knows more about that should probably decide :)

**Comment by martinpitt** [comment]
> pin the old version

Done, that's the safest and most sensible. Verified locally.

**Review by martinpitt — APPROVED** [review]

File: build.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-import child_process from 'child_process';
-import fs from 'fs';
-import os from 'os';
-import path from 'path';
-import process from 'process';
+import child_process from 'node:child_process';
+import fs from 'node:fs';
+import os from 'node:os';
+import path from 'node:path';
+import process from 'node:process';
```
Comment (jelly) [inline build.js:7]: Is this a standard anywhere? Or something deno invented? Please link the upstream documentation at least. So is this supported for all our supported nodejs versions? 18 and up?

https://nodejs.org/api/esm.html#node-imports

What is the benefit of these imports however?

File: package.json [hunk 2]
```diff
@@ -46,13 +46,16 @@
     "eslint-plugin-react": "7.34.3",
     "eslint-plugin-react-hooks": "4.6.2",
     "gettext-parser": "8.0.0",
+    "glob": "7.2.3",
     "htmlparser": "1.7.7",
     "jed": "1.1.1",
+    "lodash": "4.17.21",
```
Comment (jelly) [inline package.json:52]: This is only used for:
```
                const output = _.template(data.toString())({
                    title: test,
                    builddir: file.split("/").map(() => "../").join(""),
                    script: test + '.js',
                });
```

Time for a clever `replace()`.?

Comment (jelly) [reply package.json:52]:
```
[jelle@t14s][~/projects/cockpit/main]%du -hs node_modules/lodash
4.9M    node_modules/lodash
```

But ofcourse react-tables uses it.

File: build.js (hunk 1 above)
Comment (martinpitt) [reply build.js:7]: We already did *some* imports with `node:` prefixes before, just not consistently. E.g. here:

> pkg/lib/esbuild-compress-plugin.js:import child_process from 'node:child_process';

That's how they are [documented](https://nodejs.org/api/child_process.html). Your link says the same.

File: package.json (hunk 2 above)
Comment (martinpitt) [reply package.json:52]: That's a nice idea though, then we at least lose the dependabot noise.

Comment (jelly) [reply package.json:52]: Or alternative https://www.npmjs.com/package/lodash.template

Comment (jelly) [reply package.json:52]: tbh lodash doesn't seem to get updates :)

Comment (allisonkarlitskaya) [reply package.json:52]: I like lodash and always think that I should use it more, but usually forget.  :)

**Pull request merged**